### DISCUSSION TITLE
How to disable media keys?

### DISCUSSION THREAD

**Discussion by toppits** [quodlibet/quodlibet#4246] [Q&A] [answered]
Kubuntu, but cant seem to find a way to make quodlibet not use the media keys

**Comment by jej** [comment, marked as answer]
According to the source code, you can disable it with the `advanced_preferences` plugin:

https://github.com/quodlibet/quodlibet/blob/fb65cb3d7126c8dd9023d6acf3641c99f41bf87c/quodlibet/ext/events/advanced_preferences.py#L170-L173

Not tested. Hope it helps.
J.

**Reply by toppits** [reply]
yep, that seems to do it. Thank you
